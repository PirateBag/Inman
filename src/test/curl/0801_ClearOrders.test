```Purpose
Clear the contents of the following tables:
  OrderLineItems (Only)
```
==JSON==
{ "idToSearchFor" : "2" }

