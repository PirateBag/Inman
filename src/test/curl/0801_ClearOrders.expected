{
  "responseType" : null,
  "errors" : [ ],
  "data" : [ {
    "id" : 0,
    "crudAction" : "NONE",
    "message" : "OrdereLineItems deleted"
  }, {
    "id" : 0,
    "crudAction" : "NONE",
    "message" : "Order Line Items PK reset"
  } ]
}